ci/gpu/build.sh

```diff
@@ -38,11 +38,14 @@ logger "Check versions..."
 python --version
 gcc --version
 g++ --version
-conda list
 
 # FIX Added to deal with Anancoda SSL verification issues during conda builds
 conda config --set ssl_verify False
 
+conda install -c rapidsai/label/cuda$CUDA_REL -c rapidsai-nightly/label/cuda$CUDA_REL -c nvidia/label/cuda$CUDA_REL -c conda-forge \
```

Comment:
These are defined in the `.condarc` of the gpuci container.
```suggestion
conda install \
```